src/coredata.cs

```diff
@@ -1136,6 +1136,34 @@ interface NSManagedObjectContext : NSCoding
 		[Watch (4, 0), TV (11, 0), Mac (10, 13), iOS (11, 0)]
 		[NullAllowed, Export ("transactionAuthor")]
 		string TransactionAuthor { get; set; }
+
+		[Notification (typeof (NSManagedObjectsIdsChangedEventArgs))]
+		[Watch (7,0), TV (14,0), Mac (11,0), iOS (14,0)]
+		[Field ("NSManagedObjectContextDidMergeChangesObjectIDsNotification")]
+		NSString DidMergeChangesObjectIdsNotification { get; }
+
+		[Notification (typeof (NSManagedObjectsIdsChangedEventArgs))]
+		[Watch (7,0), TV (14,0), Mac (11,0), iOS (14,0)]
+		[Field ("NSManagedObjectContextDidSaveObjectIDsNotification")]
+		NSString DidSaveObjectIdsNotification { get; }
+	}
+
+	[Watch (7,0), TV (14,0), Mac (11,0), iOS (14,0)]
+	interface NSManagedObjectsIdsChangedEventArgs {
+		[Export ("NSDeletedObjectIDsKey")]
+		NSSet DeletedObjectIds { get; }
```

Comment:
@mandel-macaque Why doesn't this one get a `Key` suffix when all the other members here do?